packages/dev/core/src/Cameras/Inputs/freeCameraTouchInput.ts

```diff
@@ -76,7 +79,7 @@ export class FreeCameraTouchInput implements ICameraInput<FreeCamera> {
             this._pointerInput = (p) => {
                 const evt = <IPointerEvent>p.event;
 
-                const isMouseEvent = evt.pointerType === "mouse";
+                const isMouseEvent = evt.pointerType === "mouse" || (this._isSafari && typeof evt.pointerType === "undefined");

```

Comment:
Would that not break on mobile ??? can you try on ios ?

Reply:
I tested things on my iPad and it's working as expected.  In addition, `Tools.GetPointerPrefix()` evaluates to "pointer" as well as has the correct values for `pointerType` so this shouldn't break iOS.

Reply:
Do you have an ios device to check ? if not I could try locally on one ? better safe than sorry with Safari :-)

Reply:
> Do you have an ios phone device to check ? if not I could try locally on one ? better safe than sorry with Safari :-)

Reply:
Yup, everything works on my iPhone.  I even verified the values for Tools.GetPointerPrefix and that pointer events generated on my iPhone have a correct value for pointerType.  We should be good with iOS